CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased - expected 2020-09-20
 ### Added
 ### Changed
+- It is now an error if a macro is configured with 0 operations.
```

Comment:
It'd be nice to have a test for this, because it's a user-visible behaviour and we don't want to change it unwittingly.

Reply:
Done and squashed into existing commit: https://github.com/newsboat/newsboat/pull/1134/commits/451f1b5e00d5623334260c08e2aa24aa52b53feb#diff-64be1adcbd17050bc44ab8b1eea4771aR218